src/hooks/screen/useChatScreenHook.ts

```diff
@@ -28,12 +28,11 @@ export const ScrollContext = React.createContext<ScrollContextProps | null>(null
 
 function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
   const {localDb, chat, refresh} = useLocalDatabaseHook();
-  const {selectedChannel, goBackFromChatScreen, goToChatInfoScreen} = useChatUtilsHook();
+  const {selectedChannel, goBackFromChatScreen, goToChatInfoScreen} = useChatUtilsHook(type);
   const {anonProfileId} = useUserAuthHook();
 
   const [selfAnonUserInfo, setSelfAnonUserInfo] = React.useState<any>(null);
   const [chats, setChats] = React.useState<ChatSchema[]>([]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useChatUtilsHook` now accepts a type parameter, which is a good change as it allows for more flexible usage of the hook. However, I noticed that the `selfAnonUserInfo` state variable is initialized with `null` and its type is set to `any`. This could potentially lead to type errors in your code. It would be better to define an interface for this state variable to ensure type safety.

```diff
- const [selfAnonUserInfo, setSelfAnonUserInfo] = React.useState<any>(null);
+ interface SelfAnonUserInfo {
+   // Define the properties here
+ }
+ const [selfAnonUserInfo, setSelfAnonUserInfo] = React.useState<SelfAnonUserInfo | null>(null);
```
This way, you can ensure that `selfAnonUserInfo` always adheres to the `SelfAnonUserInfo` structure, or is `null`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->